pyproject.toml

```diff
@@ -71,26 +71,38 @@ target-version = "py38"
 
 [tool.ruff.lint]
 select = [
+  "A",      # flake8-builtins
   "AIR",    # Airflow
   "ASYNC",  # flake8-async
+  "B",      # flake8-bugbear
   "BLE",    # flake8-blind-except
+  "C4",     # flake8-comprehensions
   "C90",    # McCabe cyclomatic complexity
   "DJ",     # flake8-django
   "DTZ",    # flake8-datetimez
   "E",      # pycodestyle errors
+  "EM",     # flake8-errmsg
   "F",      # Pyflakes
   "FIX",    # flake8-fixme
   "FLY",    # flynt
   "G",      # flake8-logging-format
   "ICN",    # flake8-import-conventions
   "INP",    # flake8-no-pep420
   "INT",    # flake8-gettext
+  "ISC",    # flake8-implicit-str-concat
```

Comment:
The above lints look useful to me in that the changes made to make the lints pass mostly look like useful improvements. The possible exception to my mind is defining a `msg` variable before raising the exception - I am ambivalent to that one.